tests/yast2_gui/yast2_storage_ng.pm

```diff
@@ -0,0 +1,242 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: YaST2 ...
+# Maintainer: Paolo Stivanin <pstivanin@suse.com>
+
+use base "y2x11test";
+use strict;
+use warnings;
+use testapi;
+use version_utils qw(is_sle is_tumbleweed is_opensuse);
+
+sub add_logical_volume {
+    my ($lvname, $role) = shift;
+    sleep 1;
+    wait_screen_change { send_key "alt-a" };
+    wait_screen_change { type_string "$lvname" };
+    wait_screen_change { send_key "alt-n" };
+    # custom size
+    if (is_sle('<=12-sp4')) {
```

Comment:
`send_key(  is_sle('<=12-sp4') ? "alt-c" : "alt-c" );` - I like more but of course it is just matter of taste :)

Reply:
nice one :) I too prefer your solution. I didn't know I could use the ternary operator also in such a case :+1:

Reply:
in case you accepting it here , you may go further through your code and find few more places where you can use same pattern

Reply:
I already spotted some other places! I'll definitely change all occurrences :+1: